src/partition/src/manager.rs

```diff
@@ -246,6 +246,17 @@ impl PartitionRuleManager {
         let splitter = WriteSplitter::with_partition_rule(partition_rule);
         splitter.split_insert(req, schema)
     }
+
+    pub async fn split_delete_request(
+        &self,
+        table: TableId,
+        req: DeleteRequest,
+        primary_key_column_names: Vec<&String>,
+    ) -> Result<DeleteRequestSplit> {
+        let partition_rule = self.find_table_partition_rule(table).await.unwrap();
```

Comment:
We can simply propagate the error:

```suggestion
        let partition_rule = self.find_table_partition_rule(table).await?;
```

Reply:
Done: [5f27dd8](https://github.com/GreptimeTeam/greptimedb/pull/2150/commits/5f27dd85057ba9868eb42cd95afd63b7d2d362c2)

Please let me know if there are any errors or omissinos.